mobileu/settings.py

```diff
@@ -194,3 +194,12 @@
     from oneplusmvp.settings import OPEN_SCHOOL
 except ImportError as e:
     OPEN_SCHOOL = "Open School"
+
+# Junebug config
+JUNEBUG_BASE_URL = os.environ.get("JUNEBUG_BASE_URL", "https://junebug.qa.dig-it.me/jb/channels/")
+JUNEBUG_CHANNEL_ID = os.environ.get("JUNEBUG_CHANNEL_ID", "ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/")
```

Comment:
instead of using legit values, can you either leave these blank or use dummy data. This is a public repository after all.